cob_default_robot_config/cob4-2/head_joint_configurations.yaml

```diff
@@ -5,16 +5,16 @@ default_vel: 0.4
 
 home: [[0.0, 0.0, 0.0]]
 
-front: [[0.0, 0.0, 0.0]]
-front_down: [[1.57, -3.14, 1.57]]
-front_down_twisted: [[1.57, -3.14, -1.57]]
+front: [[0.0, 0.0, 3.14]]
```

Comment:
I also notified this error problem, the urdf has to be updated. The real hardware doesn't correspond to the current urdf model. Please keep this pull request open until we solve the problem.